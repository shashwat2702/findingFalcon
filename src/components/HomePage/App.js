import React from 'react';
import './App.scss';

function App() {
  return (
    <div className="homePage">
      HomePage
    </div>
  );
}

export default App;
