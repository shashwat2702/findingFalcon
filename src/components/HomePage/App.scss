.listOfDropDown {
    display: flex;
    justify-content: space-between;
    margin: 2rem;
    @media screen and (max-width: 768px) {
        flex-direction: column;
        align-items: center;
    }
    .planetDropDown {
        width: 300px;
        margin: 1rem;
    }
}

