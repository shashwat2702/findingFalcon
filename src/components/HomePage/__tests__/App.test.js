import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

jest.mock('react-router-dom');

describe('HomePage snapshot', () => {
  it('should render correctly', () => {
    const homePageSnap = renderer.create(<App />);
    const homePageSnapJson = homePageSnap.toJSON();
    expect(homePageSnapJson).toMatchSnapshot();
  });
});
