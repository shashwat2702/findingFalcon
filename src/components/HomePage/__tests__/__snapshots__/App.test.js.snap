// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage snapshot should render correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Edit
       
      <code>
        src/App.js
      </code>
       
      and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
